modules/install/pages/install-platforms.adoc

```diff
@@ -105,6 +105,26 @@ Follow the best practices to run xref:best-practices-vm.adoc[Couchbase Server on
 Follow the best practices to run xref:best-practices-vm.adoc[Couchbase Server on a virtualized environment].
 |===
 
+[#supported-ARM-platforms]
+== ARM Platforms
+
+.Supported ARM platforms
+|===
+| Operating system | Couchbase Version | Notes
+
+| Amazon  Linux 2 (64 bits)
+| 7.1
+| {empty}
+
+| Linux ARM
+| Not supported
+| {empty}
+
+| MacOS 11+
+| Not supported
```

Comment:
Couchbase Server is supported on MacOS Apple Silicon (ARM) when using the MacOS Rosetta2 emulation layer.